src/components/bcourses/roster/RosterPhotos.vue

```diff
@@ -38,7 +38,7 @@
           </div>
         </v-card-title>
         <v-card-text>
-          <div :id="`student-id-${student.studentId}`">
+          <div class="display-none-when-print" :id="`student-id-${student.studentId}`">
```

Comment:
@mlincoln - Looks right. Will the linter want 'id' before 'class'? We'll find out...

Reply:
It did not like that. Just pushed an amendment.